src/modules/Accordion/AccordionTitle.js

```diff
@@ -43,17 +50,29 @@ export default class AccordionTitle extends Component {
     parent: 'Accordion',
   }
 
+  static create = createShorthandFactory(AccordionTitle, content => ({ content }))
+
   handleClick = (e) => {
     const { onClick } = this.props
 
     if (onClick) onClick(e, this.props)
   }
 
+  renderContent(content) {
+    return isValidElement ? content : (
```

Comment:
The `isValidElement` is missing its argument `isValidElement(content)`, however, this can be removed entirely and the `content` always rendered in place.

Also, we try to avoid all styling and leave this to the user's CSS theme of choice.  Let's remove the extra wrapping `div` and inline style by going with two explicit returns in the [render()](url) method:

```jsx
if (!_.isNil(content)) {
  return (
    <ElementType {...rest} className={classes} onClick={this.handleClick}>
      <Icon name='dropdown' />
      {content}
    </ElementType>
  )
}

return (
  <ElementType {...rest} className={classes} onClick={this.handleClick}>
    {children}
  </ElementType>
)
```